src/sdk/namespace_cache.js

```diff
@@ -258,15 +263,21 @@ class NamespaceCache {
                             content_type: params.content_type,
                             xattr: object_info_hub.xattr,
                             last_modified_time: (new Date(object_info_hub.create_time)).getTime(),
+                            update_cache_stats: data_length =>
```

Comment:
make a small utility function:
```javascript
function update_cache_stats_hook(bucket_name) {
  return write_bytes =>this.stats_collector.update_cache_stats({ bucket_name, write_bytes });
} 
```
and then call it here liket his:
```suggestion
                            update_cache_stats: update_cache_stats_hook(params.bucket),
```